Scripts/SDL/Elonin.sdl

```diff
@@ -0,0 +1,80 @@
+# /*==LICENSE==*
+# 
+# CyanWorlds.com Engine - MMOG client, server and tools
+# Copyright (C) 2011 Cyan Worlds, Inc.
+# 
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+# 
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+# 
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+# 
+# Additional permissions under GNU GPL version 3 section 7
+# 
+# If you modify this Program, or any covered work, by linking or
+# combining it with any of RAD Game Tools Bink SDK, Autodesk 3ds Max SDK,
+# NVIDIA PhysX SDK, Microsoft DirectX SDK, OpenSSL library, Independent
+# JPEG Group JPEG library, Microsoft Windows Media SDK, or Apple QuickTime SDK
+# (or a modified version of those libraries),
+# containing parts covered by the terms of the Bink SDK EULA, 3ds Max EULA,
+# PhysX SDK EULA, DirectX SDK EULA, OpenSSL and SSLeay licenses, IJG
+# JPEG Library README, Windows Media SDK EULA, or QuickTime SDK EULA, the
+# licensors of this Program grant you additional
+# permission to convey the resulting work. Corresponding Source for a
+# non-source form of such a combination shall include the source code for
+# the parts of OpenSSL and IJG JPEG Library used as well as that of the covered
+# work.
+# 
+# You can contact Cyan Worlds, Inc. by email legal@cyan.com
+#  or by snail mail at:
+#       Cyan Worlds, Inc.
+#       14617 N Newport Hwy
+#       Mead, WA   99021
+# 
+# *==LICENSE==*/
+ 
+STATEDESC Elonin
+ 
+{
+   VERSION 1
+ 
+    VAR BOOL    fntOnOff[1] DEFAULT=0
+    VAR BOOL     birdie[1] DEFAULT=0
+    VAR INT     flwrCorrectCount[1] DEFAULT=0
+    VAR BOOL    flwrPuzSolved[1] DEFAULT=0
+    VAR BOOL    flwrPuzInUse[1] DEFAULT=0
+    VAR INT     keyPzlCorrectPushes[1] DEFAULT=0
+    VAR BOOL    keyPzlSolved[1] DEFAULT=0
+    VAR BOOL    keyPzlInUse[1] DEFAULT=0
+    VAR BOOL    fireOne[1] DEFAULT=0
+    VAR BOOL    fireTwo[1] DEFAULT=0
+ 
+ 
+}
```

Comment:
Is this version used on any production shards? If not, I think we should omit it.

Reply:
> Is this version used on any production shards? If not, I think we should omit it.

This is the exact version used on Minkata/MOULa. /shrug

Reply:
So version 2 is unused?

Reply:
> So version 2 is unused?

I'm not sure. Just carrying over what's on OU, FWIW. Ametist would know more about the finer details.

Reply:
I don't know what you mean...I have in Elonin.sdl two versions, the latest VERSION 2 has two more variables than version 1.

Reply:
I would like to avoid accumulating "development churn" into mainline. If version 2 was the first SDL used on production shards, (eg Gehn or MOULa), then I think we should consider omitting version 1 in the repository. In that way, we don't accumulate more SDL files with 50+ versions that have to be scrolled through for a shard admin to figure out what's going on.

Reply:
> I would like to avoid accumulating "development churn" into mainline. If version 2 was the first SDL used on production shards, (eg Gehn or MOULa), then I think we should consider omitting version 1 in the repository. In that way, we don't accumulate more SDL files with 50+ versions that have to be scrolled through for a shard admin to figure out what's going on.

Fair enough. I just committed a new version of the SDL with `VERSION 2` as `VERSION 1`.

Reply:
Won't this cause problems if used on MOULa? The version there should be updated as well to avoid conflicts.

Reply:
> Won't this cause problems if used on MOULa? The version there should be updated as well to avoid conflicts.

To answer this question and also elaborate on Adam's comment regarding the state blobs on TrollLand... yes, this change will cause major problems if it is used on MOULa. You cannot re-number a version of an SDL as @DoobesURU did with version 2 becoming version 1. What Adam was suggesting originally was to omit the version 1 block entirely and keep only version 2 **as version 2**.

Please do not take this PR with your self-built clients to MOULa.